### PR TITLE
Retire Old Weather: Whaling

### LINKED ISSUE #8: Archive page for whaling.oldweather.org
Just copying here the thread from Slack, so it's easy to retrieve and Zach, Adam, Sarah can discuss as needed. 

Laura:
whaling.oldweather.org is officially no longer active, as of last week. They finished the last data set and now volunteers are only working through the spreadsheet effort (via links within oldweather.org).

Kevin’s request was that we move whaling.oldweather.org to x.oldweather.org for a couple months, if possible, while he goes through all the materials and moves over remaining stuff to his archives, etc.

But no active classifying, etc. is happening at whaling.oldweather.org (not sure if that helps you at all).

(Apparently moving to x.oldweather.org is something [Adam?] did for classic.oldweather.org previously, so they could do the same look-through of materials before fully retiring it, but also make sure that no volunteers went to the old URL (e.g. whaling.oldweather.org) and wasted time trying to do anything there.

Adam asked:
What do we want to put up in its place? Just the normal retirement page (similar to https://whale.fm)?

Laura responded:
Yes, thank you for thinking of that. A retirement/archived page along those lines would be good. Would it be helpful if I provide you copy for that?

Adam:
That'd be great, thanks. I was thinking this might be a good thing for @zach to do if he has time since it involves touching a couple of different parts of the infrastructure (DNS and Nginx config)

Zach:
:+1:

Zach:
link me at an issue with the details and I’ll put it on my ops list

Adam:
I'd be happy to pair on it next week too if you'd like

Zach:
Sure, keep in mind that I’ll be in Oxford the week of the 8th too, if it can wait.

### PR DESCRIPTION
**Pull request by wgranger**
Creates folder and adds content for Old Weather: Whaling retirement.

Closes #8

### CODE REVIEW

File: OldWeatherWhaling/index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8" lang=""> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9" lang=""> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js" lang=""> <!--<![endif]-->
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <title>Old Weather: Whaling: Archive</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="apple-touch-icon" href="apple-touch-icon.png">
```
Comment (srallen) [inline OldWeatherWhaling/index.html]: This is just a placeholder for the web. This can be removed.

File: OldWeatherWhaling/index.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+      <div class="container">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="#">Archived Zooniverse Project: Old Weather: Whaling</a>
```
Comment (srallen) [inline OldWeatherWhaling/index.html]: Maybe just make this a span

File: OldWeatherWhaling/index.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        <script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
+    </head>
+    <body>
+        <!--[if lt IE 8]>
+            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+      <div class="container">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
```
Comment (srallen) [inline OldWeatherWhaling/index.html]: Remove this. On mobile it's showing a hamburger menu that doesn't work.

File: OldWeatherWhaling/index.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="#">Archived Zooniverse Project: Old Weather: Whaling</a>
+        </div>
+        <div id="navbar" class="navbar-collapse collapse">
```
Comment (srallen) [inline OldWeatherWhaling/index.html]: Also remove this.

File: OldWeatherWhaling/index.html [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+          <h1>Old Weather: Whaling</h1>
+          <p>Whalers have sought their quarry in perilous Arctic waters for centuries. Through Old Weather: Whaling, Zooniverse citizen scientists marked and transcribed ships' logbooks, searching for new clues about climate and history.</p>
+          <p><a class="btn btn-primary btn-lg" href="https://www.oldweather.org" role="button">Old Weather &raquo;</a></p>
+          <p><a class="btn btn-primary btn-lg" href="https://www.zooniverse.org/publications" role="button">Publications &raquo;</a></p>
+          <p><a class="btn btn-primary btn-lg" href="https://www.zooniverse.org/" role="button">Zooniverse &raquo;</a></p>
+        </div>
+      </div>
+    </div>
+
+    <div class="container">
```
Comment (srallen) [inline OldWeatherWhaling/index.html]: Remove this.

File: OldWeatherWhaling/css/main.css [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+	border-radius: 10px;
+}
+
+img.cover {
+	height: 400px;
+	width: 400px;
+	object-fit: cover;
+}
+
+img.similar {
```
Comment (srallen) [inline OldWeatherWhaling/css/main.css]: The styles starting on this line and down aren't being used and can be removed.

**Review by srallen — CHANGES_REQUESTED** [review]
Looks like some files maybe used on the other static retirement pages got copied into this folder. The other project images in `/img` as well as the pngs for the apple and microsoft tiles and the xml file can be removed.

Also let's set the background color for the body to what the jumbotron section is using, `#eee` so it doesn't look like there's missing content.

**Comment by wgranger** [comment]
Thanks for the keen eye on the review. I also added some css to make the page responsive at mobile widths.

**Review by srallen — APPROVED** [review]

**Comment by wgranger** [comment]
Are we waiting on anything else for this, or can it be merged?

**Pull request merged**